build/build/src/project/gui2.rs

```diff
@@ -125,6 +125,32 @@ impl IsTarget for Gui2 {
         let WithDestination { inner: _, destination } = job;
         async move {
             let repo_root = &context.repo_root;
+            crate::ide::web::google_font::download_google_font_with_css(
+                &context.cache,
+                &context.octocrab,
+                "mplus1",
+                "M PLUS 1",
+                "/font-mplus1",
+                repo_root.app.gui_2.public.font_mplus_1.to_path_buf(),
+                repo_root.app.gui_2.src.assets.font_mplus_1_css.to_path_buf(),
```

Comment:
Do we need `to_path_buf` if we are awaiting?
Perhaps just `&repo_root.app.gui_2.public.font_mplus_1` is enough?